src/etf2l/services/etf2l-api.service.ts

```diff
@@ -22,6 +23,15 @@ export class Etf2lApiService {
     const url = `${etf2lApiEndpoint}/player/${steamIdOrEtf2lId}`;
     return await firstValueFrom(
       this.httpService.get<Etf2lPlayerResponse>(url).pipe(
+        catchError((err: AxiosError) => {
+          if (err.message === 'Request failed with status code 404') {
```

Comment:
How about
```ts
if (err.response?.status === 404)
```
You can also use `HttpStatusCode.NotFound` instead of 404.

Reply:
changed